doc/run_experiment.rst

```diff
@@ -882,6 +913,17 @@ predictions.
     :ref:`log <log>`, :ref:`models <models>`, and
     :ref:`predictions <predictions>`.
 
+
+.. _save_cv_folds:
+
+save_cv_folds *(Optional)*
+""""""""""""""""""""""""""
+
```

Comment:
Which loop folds would these be?

Reply:
Whatever folds end up being used.

Reply:
So if I have a `cross-validate` task, would these be outer or inner loop?

Reply:
This is always the outer loop - the ones referred to by `cv_folds`.